playbooks/group_vars/cloudhost.yml

```diff
@@ -3,6 +3,7 @@
 ## Base Vars
 nexcess_repo_enabled: false
 server_selinux_enforcing: false
+mysql_pid_file: /var/lib/mysql
```

Comment:
mysql_pid_file should be the full path to the pid file, not a directory. mysql_pid_file is being used directly as pid-file in the configuration in the role here: https://github.com/nexcess/ansible-role-mariadb/blob/5d7866e4cac5dc83a18c6fdf73d5ce2ca6ef8e01/templates/my.cnf.j2#L18